targets/ar71xx-mikrotik

```diff
@@ -4,8 +4,16 @@ config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
 # ath5k cards are commonly used with Mikrotik hardware
 packages 'kmod-ath5k'
 
+if [ "$GLUON_WLAN_MESH" = '11s' ]; then
+device mikrotik-mantbox-15s-r2 nand-large
```

Comment:
The first parameter to device is the name used by the autoupdater. To discover the name that should go there run the following command on your preliminary image:

```
lua -e 'print(require("platform_info").get_image_name())'
```